EIPS/eip-5453.md

```diff
@@ -0,0 +1,54 @@
+---
+eip: 5453
+title: Smart Endorsement
+description: Defines a format for endorsement on-chain
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-5453-endorsement-standard/10355
+status: Draft
+type: Standards Track
+category: Interface
+created: 2022-08-12
+requires: 5269
+---
+
+## Abstract
+
+Provides a format for endorsement of smart contract transactions in the format of extra data (the last bytes field of a method signature).
+
+## Motivation
+
+1. Support a second approval from another user: TODO
+2. Support pay-for-by: TODo
+3. Support multi-sig: TODO
+4. Support persons acting in concert by endorsements: TODO
+5. Support accumulated voting: TODO
+6. Support off-line signatures: TODO
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+
+```
+
+## Rationale
+
+1. Choosing [EIP-5269](./eip-5269.md) instead of [EIP-165](./eip-165.md) because the data format cannot be represneted by format.
```

Comment:
```suggestion
1. Choosing [EIP-5269](./eip-5269.md) instead of [EIP-165](./eip-165.md) because the data format cannot be represented by format.
```
This is unclear. You're also referencing something that doesn't exist.

Reply:
It now does

Reply:
The typo still needs fixing though.